python/mxnet/_numpy_op_doc.py

```diff
@@ -748,3 +748,72 @@ def _np_moveaxis(a, source, destination):
     (5, 4, 3)
     """
     pass
+
+def _np__random_shuffle(x):
+    """
+    Modify a sequence in-place by shuffling its contents.
+
+    This function only shuffles the array along the first axis of a
+    multi-dimensional array. The order of sub-arrays is changed but
+    their contents remains the same.
+
+    Parameters
+    ----------
+    x: ndarray
+    The array or list to be shuffled.
+
+    Returns
+    -------
+        None
+
+    Examples
+    --------
+    >>> arr = np.arange(10)
+    >>> np.random.shuffle(arr)
+    >>> arr
+    array([5., 1., 0., 6., 7., 3., 9., 8., 4., 2.])  # random
+
+    Multi-dimensional arrays are only shuffled along the first axis:
+
+    >>> arr = np.arange(9).reshape((3, 3))
+    >>> np.random.shuffle(arr)
+    >>> arr
+    array([[6., 7., 8.], # random
+            [3., 4., 5.],
+            [0., 1., 2.]])
+    """
+    pass
+
+def _np_broadcast_to(array, shape=[], out=None):
```

Comment:
According to NumPy doc, should be
```python
def _np_broadcast_to(array, shape, out=None)
```
https://docs.scipy.org/doc/numpy/reference/generated/numpy.broadcast_to.html#numpy.broadcast_to